packages/ra-core/src/form/getSimpleValidationResolver.ts

```diff
@@ -30,15 +44,16 @@ export const getSimpleValidationResolver = (validate: ValidateForm) => async (
     if (!errors || Object.getOwnPropertyNames(errors).length === 0) {
         return { values: data, errors: {} };
     }
+    const flattenErrors = flattenKeys(errors);
```

Comment:
I'd name the variable `flattenedErrors` (otherwise it starts with a verb and looks like a callback)